### ISSUE TITLE
Returned console.table() is broken

### ISSUE THREAD

**Issue by WuglyakBolgoink** [peteringram0/babel-plugin-console-source#2]
before/without plugin:
```
console.table(information);
```

🚫 after plugin:
```
console.table("details/information-controller.js (2044:12)", information);
```

✅ should be:
```
console.table(information);
```

See: https://developer.mozilla.org/de/docs/Web/API/Console/table

**Comment by peteringram0** [comment]
I'll have a look at this in the next few days for you.

**Comment by peteringram0** [comment]
This has been fixed in version 2.0.2 (on NPM)

https://github.com/peteringram0/babel-plugin-console-source/commit/ada9f08870cf8b7b2461eba3fb38a0d483b1f773

**Comment by WuglyakBolgoink** [comment]
@peteringram0 Thank you!
